container/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM python:3.9.12-slim-buster
```

Comment:
Could you add a section in README.md about "How to build and run docker-image for artifactory-cleanup"?